api/apps/articles/model/save.coffee

```diff
@@ -23,7 +23,7 @@ cloneDeep = require 'lodash.clonedeep'
 EMBEDLY_KEY, FORCE_URL, ARTSY_EDITORIAL_ID, SECURE_IMAGES_URL } = process.env
 sailthru = require('sailthru-client').createSailthruClient(SAILTHRU_KEY,SAILTHRU_SECRET)
 { crop } = require('embedly-view-helpers')(EMBEDLY_KEY)
-artsyXapp = require('artsy-xapp')
+artsyXapp = require('artsy-xapp').token or ''
```

Comment:
Yeah, do you mean like [this?](https://github.com/artsy/positron/blob/master/api/apps/articles/test/model/save.coffee#L25). I'm probably rewiring it incorrectly, but this works intermittently